templates/styled-system/tests/__snapshots__/snapshots.js.snap

```diff
@@ -0,0 +1,13 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Box renders 1`] = `null`;
```

Comment:
kit-snapshot currently only accepts elements but these should render once it supports components as well